frontend/src/containers/Addgroup.js

```diff
@@ -0,0 +1,392 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import LoginTopBar from '../containers/LoginTopBar';
+import Notification from '../containers/Notification';
+
+import CustomTextArea from '../components/CustomTextArea';
+import ImagePicker from '../components/ImagePicker';
+import Input from '../components/CustomTextArea';
+import PublicFooter from '../components/PublicFooter';
+import Select from '../components/Select';
+
+import appendGroupForm from '../actions/form/append_group';
+import createGroup from '../actions/groups/create';
+import notify from '../actions/notification/notify';
+import uploadImage from '../actions/images/upload';
+import validateSchema from '../actions/form/validate_schema';
+
+import groupSchema from '../joi_schemas/group';
+
+import i18n from '../lib/i18n';
+
+const memberRole = 'MEMBER';
```

Comment:
I've grown accustomed to having `ALL_CAPS` for constants, however I realize that was a pattern intended for es5, what do think?